src/pages/VenuePage/VenuePage.tsx

```diff
@@ -301,15 +300,13 @@ const VenuePage = () => {
     case VenueTemplate.friendship:
       template = <FriendShipPage />;
       break;
+    case VenueTemplate.themecamp:
     case VenueTemplate.partymap:
       template = <PartyMapRouter />;
       break;
     case VenueTemplate.artpiece:
       template = <ArtPiece />;
       break;
-    case VenueTemplate.themecamp:
-      template = <CampRouter />;
-      break;
```

Comment:
💥 🎉 !!

Reply:
I would say OOS/Future. We gain everything we need in the short term without having to touch data migrations here. Then we can do data migrations at our leisure without blocking these changes. Then we just have a tiny ~1 line PR to remove the then redundant overload.

Reply:
backlogged issue to explore tools/patterns for neatly handling data migrations in https://github.com/sparkletown/internal-sparkle-issues/issues/152#issuecomment-758547067